packages/shylu/shylu_node/basker/test/amesos2_interface_coverage_test.cpp

```diff
@@ -74,8 +74,8 @@ int main(int argc, char* argv[])
   
   //Starting up Kokkos
   int nthreads = 4; // We will not use all 4 in all tests
-  Kokkos::InitArguments init_args;
-  init_args.num_threads = nthreads;
+  Kokkos::InitializationSettings init_args;
+  init_args.set_num_threads(nthreads);
```

Comment:
Consider doing `Kokkos::initialize(Kokkos::InitializationSettings().set_num_threads(4));`

Reply:
> The use of `volatile` in https://github.com/trilinos/Trilinos/blob/kokkos-promotion/packages/ifpack2/src/Ifpack2_Relaxation_def.hpp#L859 causes a kokkos deprecation warning in SPARC. Any chance we can remove it in this PR?

@jewatkins thanks for the feedback, and yes, I'm working on cleaning this will push the changes up soon